src/common/models/Projection.ts

```diff
@@ -14,6 +14,36 @@ import {
 import { ICUHeadroomInfo, calcICUHeadroom } from './ICUHeadroom';
 import { lastValue } from './utils';
 
+const DISABLED_CASE_DENSITY: string[] = [
+  '48029', // https://trello.com/c/ulCnCcif/
+];
+
+const DISABLED_INFECTION_RATE: string[] = [];
+
+const DISABLED_TEST_POSITIVITY: string[] = [
+  '25001',
+  '25003',
+  '25005',
+  '25007',
+  '25009',
+  '25011',
+  '25013',
+  '25015',
+  '25017',
+  '25019',
+  '25021',
+  '25023',
+  '25025',
+  '25027',
+  '48113',
+  '48215',
```

Comment:
All of the above were preexisting disablements.  I just moved them up here to a top-level constant.